app/views/person/security_tools/index.html.haml

```diff
@@ -76,6 +99,22 @@
       = @suspend_person_situation_text
     %div.well
       = @suspend_person_solution_text
+
+    - if person.blocked?
+      %strong.bigger-title
+        = @blocked_person_situation_text
+      %div.well
+        = @blocked_person_solution_text
```

Comment:
@carlobeltrame in den TechSpec sagst du, dass ein zusätzlicher Text angezeigt werden soll, wenn die beiden Fristen in den Settings gesetzt sind. Kann das mit einem custom content abgedeckt werden oder soll ich mehrere custom_contents anlegen? Das habe ich noch nicht so durchschaut

Reply:
Ich würde hier eigentlich einfach einen I18n text anzeigen, es gibt ja keine Anforderung, dass die Texte in der Applikation angepasst werden können müssen.
Wie ich sehe, sind aber die bestehenden Texte ebenfalls schon als `CustomContent` implementiert, drum passt das hier wohl schon.
Ja da brauchts schon mehrere CustomContent wegen der condition, aber das hast  du ja bereits so umgesetzt.

Reply:
Habe ich so umgesetzt.